### DISCUSSION TITLE
Run Pandoc script on Current Location for currently active note

### DISCUSSION THREAD

**Discussion by ReaderGuy42** [Taitava/obsidian-shellcommands#305] [Q&A] [answered]
I use a terminal for a pandoc conversion, because the Pandoc plugin is unfortunately a little hit or miss, which I can't seem to pinpoint exactly why it’s not working. So I just use the terminal to do the conversion. So I right click the Obsidian note I want to convert and hit “Show in system explorer” and there then “Open in Terminal”. I would like to use this plugin to skip this last step.

I would like the plugin to select the currently active plugin, and run this script on it:

```
pandoc {{currently_active_file}} -o {{SAME_FILeNAME}}.tex --template=/home/ReaderGuy/.pandoc/templates/default.tex --lua-filter=/home/ReaderGuy/.pandoc/highlight.lua --lua-filter=/home/ReaderGuy/.pandoc/Span.lua 
```

So far, I've figured out that I need to start the command in the plugin with `gnome-terminal` as I'm using **Linux Mint.**

But I cannot figure out how to customize the command to run on the specific file location and with the currently active note?

Any advice?
Thanks!

**Comment by ReaderGuy42** [comment, marked as answer]
After posting, I searched for something else, and found this thread in the discussions: https://github.com/Taitava/obsidian-shellcommands/discussions/280
So I solved it like that.
`pandoc {{file_path:absolute}} -o {{folder_path:absolute}}/{{title}}.tex`

**Reply by Taitava** [reply]
Good that you found an answer! 🙂
